Altis_Life.Altis/core/admin/fn_adminMenu.sqf

```diff
@@ -27,17 +29,19 @@ switch (FETCH_CONST(life_adminlevel)) do
 lbClear _list;
 
 {
-    _side = switch (side _x) do {case west: {"Cop"}; case civilian: {"Civ"}; case independent: {"Medic"}; default {"Unknown"};};
-    _list lbAdd format ["%1 - %2", _x getVariable ["realname",name _x],_side];
-    _list lbSetdata [(lbSize _list)-1,str(_x)];
-} forEach playableUnits;
-if (FETCH_CONST(life_adminlevel) < 1) exitWith {closeDialog 0;};
+    private _side = [_x] call life_util_fnc_sideToString;
+    _list lbAdd format ["%1 - %2", _x getVariable ["realname",name _x], _side];
+    _list lbSetdata [(lbSize _list)-1, str _x];
+
+    true
+} count playableUnits;
```

Comment:
if you need to add `true` to the end of count-loop, use foreach instead
https://community.bistudio.com/wiki/Code_Optimisation#forEach_vs_count_vs_findIf